### ISSUE TITLE
Koa proxy requestThunk is not a function error

### ISSUE THREAD

**Issue by yiziz** [davezuko/react-redux-starter-kit#605]
Related to https://github.com/denys/co-request/issues/22.  I noticed in `package.json` that we are using a version of `co-request` that is > `^0.2.0` that is being used in https://github.com/popomore/koa-proxy/blob/master/package.json.  This is giving `requestThunk is not a function` errors.

**Comment by davezuko** [comment]
Thanks for the issue, I don't personally make use of the middleware and without tests it tends to slip by, unfortunately. I took a look back at the original PR you submitted, and why the code was copied, and I think I have a fix for this where we don't need to copy `koa-proxy` locally and maintain the dependencies ourselves. I tested it out locally and it _seems_ to be working, but if you'd like to confirm that I'm not breaking something else you accounted for that would be awesome.

https://github.com/davezuko/react-redux-starter-kit/pull/607
